tests/base/sim/test_FairVolumeList.cxx

```diff
@@ -0,0 +1,49 @@
+/********************************************************************************
+ *   Copyright (C) 2024 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH     *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+
+#include <FairVolume.h>
+#include <FairVolumeList.h>
```

Comment:
Technically, `""` means just to prepend the local directory of the current file to the search path. This effect is clearly not used.

Repurposing the meaning of `""` to *project-local* is slightly dangerous, because it actually has a different meaning, but usually is fine.

In this case, I am using FairRoot headers from the test suite, which is technically the same as any other user. So, in my thinking, this is one place where I would like to see how the FairRoot API looks like from a user's perspective.

Furthermore, I would also like to apply this to the FairRoot library code itself, e.g. if a GeoBase file includes a Base header, it should use `<>`, but if it includes a GeoBase header it should use `""`. In other words, I would prefer a meaning of *target-local* or *library-local* instead of *project-local*.

What is the benefit of *project-local*?

In any case, I would love to see this defined as part of our coding guidelines.

Reply:
> In any case, I would love to see this defined as part of our coding guidelines (whatever definition we agree on)

:+1: 

Let's move this discussion somewhere else. I am okay with whatever you like for this PR. :-)

Resolving.